src/generators/css-filter.ts

```diff
@@ -5,12 +5,22 @@ import {parseArray, formatArray} from '../utils/text';
 import {compareURLPatterns, isURLInList} from '../utils/url';
 import {createTextStyle} from './text-style';
 import {FilterConfig, InversionFix} from '../definitions';
+import {compareChromeVersions, getChromeVersion, isChromiumBased} from '../utils/platform';
 
 export enum FilterMode {
     light = 0,
     dark = 1
 }
 
+export function hasNewBehaviorRootElements() { // https://chromium-review.googlesource.com/c/chromium/src/+/1979258 & https://bugs.chromium.org/p/chromium/issues/detail?id=501582
```

Comment:
The `new behavior` can become old some day. And other browsers are not affected. If you will have no ideas, how to give a descriptive name, about what is the behavior, let's name it like `hasChromiumIssue501582`.

Also, let's put a comment on a separate line using `// link` or `/**\n * link\n */`.